index.html

```diff
@@ -175,7 +208,12 @@ <h1 class="about-heading">About Myself</h1>
               alt="twitter icon"></a></li>
         <li><a href="mailto:khawaninalizada21@gmail.com"><img src="./Images/gmail.png" alt="Mail icon"></a></li>
       </ul>
-      <div><button class="resume">Get my resume</button></div>
+
+      <div><button class="resume" type="submit">Get my resume</button></div>
```

Comment:
Please also update the style of this button to match the other buttons in your webpage 🤗

Yoru design:

![image](https://user-images.githubusercontent.com/73714615/188138414-727bcf1e-2437-471c-b60a-b6c72a696b90.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/188138457-731854d9-c204-4f11-8c69-7af892a20c7c.png)